src/app/lib/__snapshots__/globalStyles.test.js.snap

```diff
@@ -15,66 +15,13 @@ Array [
   ],
-  "
-  @font-face {
-    font-family: \\"ReithSerif\\";
-    src: url(\\"https://gel.files.bbci.co.uk/r2.302/BBCReithSerif_W_Md.woff2\\") format(\\"woff2\\"), url(\\"https://gel.files.bbci.co.uk/r2.302/BBCReithSerif_W_Md.woff\\") format(\\"woff\\");
-    font-weight: 500;
-    font-display: optional;
-  }",
-  "
-  @font-face {
-    font-family: \\"ReithSerif\\";
-    src: url(\\"https://gel.files.bbci.co.uk/r2.302/BBCReithSerif_W_MdIt.woff2\\") format(\\"woff2\\"), url(\\"https://gel.files.bbci.co.uk/r2.302/BBCReithSerif_W_MdIt.woff\\") format(\\"woff\\");
-    font-weight: 500;
-    font-style: italic;
-    font-display: optional;
-  }",
-  "
-  @font-face {
-    font-family: \\"ReithSans\\";
-    src: url(\\"https://gel.files.bbci.co.uk/r2.302/BBCReithSans_W_Rg.woff2\\") format(\\"woff2\\"), url(\\"https://gel.files.bbci.co.uk/r2.302/BBCReithSans_W_Rg.woff\\") format(\\"woff\\");
-    font-display: optional;
-  }",
-  "
-    @font-face {
-    font-family: \\"ReithSans\\";
-    src: url(\\"https://gel.files.bbci.co.uk/r2.302/BBCReithSans_W_It.woff2\\") format(\\"woff2\\"), url(\\"https://gel.files.bbci.co.uk/r2.302/BBCReithSans_W_It.woff\\") format(\\"woff\\");
-    font-style: italic;
-    font-display: optional;
-  }",
-  "
-  @font-face {
-    font-family: \\"ReithSans\\";
-    src: url(\\"https://gel.files.bbci.co.uk/r2.302/BBCReithSans_W_Bd.woff2\\") format(\\"woff2\\"), url(\\"https://gel.files.bbci.co.uk/r2.302/BBCReithSans_W_Bd.woff\\") format(\\"woff\\");
-    font-weight: 700;
-    font-display: optional;
-  }",
-  "
-  @font-face {
-    font-family: \\"ReithSans\\";
-    src: url(\\"https://gel.files.bbci.co.uk/r2.302/BBCReithSans_W_BdIt.woff2\\") format(\\"woff2\\"), url(\\"https://gel.files.bbci.co.uk/r2.302/BBCReithSans_W_BdIt.woff\\") format(\\"woff\\");
-    font-weight: 700;
-    font-style: italic;
-    font-display: optional;
-  }",
+  [Function],
```

Comment:
Can we get some better test coverage for this file please? Weve lost the snap coverage so its not actually testing it atm